frontend/config/environment.js

```diff
@@ -60,6 +60,7 @@ module.exports = function (environment) {
       userinfoEndpoint: "/userinfo",
       afterLogoutUri: "/sso-login",
     },
+    docsBaseUrl: process.env.DOCS_BASE_URL || "https://timed.dev/docs/",
```

Comment:
```suggestion
    docsBaseUrl: process.env.DOCS_BASE_URL || "https://timed.dev/docs",
```
add the / to where it's used, makes it more readable